buildroot/share/PlatformIO/boards/BigTree_Btt002.json

```diff
@@ -14,7 +14,6 @@
         "0x3748"
       ]
     ],
-    "ldscript": "stm32f407xg.ld",
```

Comment:
These filenames were ignored in older versions of the framerwork, and the filename `ldscript.ld` was always used. The newer framework uses this filename if specified, which broke these builds. Deleting these lines should match earlier functionality.